src/libcrun/criu.c

```diff
@@ -907,7 +934,7 @@ libcrun_container_restore_linux_criu (libcrun_container_status_t *status, libcru
   ret = libcriu_wrapper->criu_set_root (root);
   if (UNLIKELY (ret != 0))
     {
-      ret = crun_make_error (err, 0, "error setting CRIU root to `%s`", root);
+      ret = crun_make_error (err, -errno, "error setting CRIU root to `%s`", root);
```

Comment:
Recursive grep in crun sources
```
$ grep -r crun_make_error . | grep ", errno," | wc -l
     477
$ grep -r crun_make_error . | grep ", -errno," | wc -l
       1
$ 
```
There is only one occurence of `-errno`. I would have expected `0`, `-ret`or `errno`.